### PR TITLE
Fix [[Class]] property of builtin constructors

### PR DESCRIPTION
**Pull request by dbatyai**
Builtin constructor functions should have "Function" as their [[Class]] property.
This fixes ch15/15.4/15.4.3/S15.4.3_A1.1_T2.js
JerryScript-DCO-1.0-Signed-off-by: Dániel Bátyai dbatyai.u-szeged@partner.samsung.com

### CODE REVIEW

File: tests/jerry/builtin-constructor-class.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Function.prototype.toString = Object.prototype.toString;
+assert(Array.toString() === "[object Function]");
+assert(Number.toString() === "[object Function]");
+assert(String.toString() === "[object Function]");
+assert(Boolean.toString() === "[object Function]");
+assert(Object.toString() === "[object Function]");
+assert(Function.toString() === "[object Function]");
+
+assert(Math.toString() === "[object Math]");
```
Comment (LaszloLango) [inline tests/jerry/builtin-constructor-class.js]: Shouldn't we add tests for RegExp and Date?

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

File: tests/jerry/builtin-constructor-class.js (hunk 1 above)
Comment (dbatyai) [inline tests/jerry/builtin-constructor-class.js]: Added

**Comment by LaszloLango** [comment]
+1 lgtm

**Comment by egavrin** [comment]
`make push`

**Comment by dbatyai** [comment]
merged: 9402e3d59c7c51b33b6a9a99f6bf7ffe77092351

**Pull request closed without merging**